.gitignore

```diff
@@ -1,7 +1,8 @@
 *.iml
 *.ipr
 *.iws
-.idea
+.idea/*
+!.idea/codeStyles
```

Comment:
Sorry can we get rid of these.

I dont want "outside opnions" on how we set up our IDEA in tis PR.

Make another PR if this is something we should do

Reply:
Sure thing! https://github.com/graphql-java/graphql-java/pull/3772/commits/535a620505004c1b92563ec0f7a2fee4f0d00f3a